### PR TITLE
Add info about using native Docker on win10 and 2016

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Describe the proposed changes

Clear up confusion about running Linux containers on Win10 and 2016

### Project version

n/a

### Related issue

n/a

### Related issue or PR in another project

https://github.com/docker/docker/issues/27668

### Please take a look

@thaJeztah @friism @londoncalling @happysysadm

### CODE REVIEW

File: docker-for-windows/index.md [hunk 1]
```diff
@@ -25,6 +25,16 @@ Please read through these topics on how to get started. To **give us your feedba
 
 >**Already have Docker for Windows?** If you already have Docker for Windows installed, and are ready to get started, skip over to the [Getting Started with Docker](/engine/getstarted/index.md) tutorial.
 
+## Using pre-installed Docker on Windows Server 2016 or Windows 10
+
+Windows Server 2016 and Windows 10 include a native Docker daemon, which allows
```
Comment (friism) [inline docker-for-windows/index.md]: @mstanleyjones I don't thinkg "include" is the best work - they still have to be installed. Maybe "Windows Server 2016 and Windows 10 can run the Docker daemon natively"

Comment (mstanleyjones) [inline docker-for-windows/index.md]: Did some rewording / reorganization to try to make it clearer. Thanks.

**Comment by londoncalling** [comment]
@mstanleyjones okay, I'm almost done with this, but let's work through the conflicts together tomorrow.

**Comment by mstanleyjones** [comment]
@londoncalling I fixed the conflicts:

1. Checked out the branch.
2. Fetched my `origin`, rebased the branch on my `origin` (to get your commit)
3. Fetched my `upstream`, rebased the branch on my `upstream` (to get the conflict)
4. Resolved the conflict in `docker-for-mac/index.md` and `docker-for-windows/index.md`) by:
    - choosing your side of the conflict
    - Fixing the formatting of the note, using indentation to make sure the list items
       were inside the note sidebar
    - Removed the unnecessary `<br>` tags
    - Wrapping the lines at 80 characters, with hanging indent, so that it's easier to
       read the source and so that in the future it's easier to maintain
    - Changing the `*` bullets to `-` ones for personal preference

File: docker-for-mac/index.md [hunk 2]
```diff
@@ -54,18 +54,22 @@ For more about stable and beta channels, see the [FAQs](faqs.md#stable-and-beta-
...
+    containers you want to keep before uninstalling the current version before
+    installing another. For more about this, see the
+    [FAQs about beta and stable channels](faqs.md#stable-and-beta-channels).
 
 ##  What to know before you install
 
-
 * **README FIRST for Docker Toolbox and Docker Machine users**: If you are already running Docker on your machine, first read [Docker for Mac vs. Docker Toolbox](docker-toolbox.md) to understand the impact of this installation on your existing setup, how to set your environment for Docker for Mac, and how the two products can coexist.
-
+<p />
```
Comment (mstanleyjones) [inline docker-for-mac/index.md]: I don't think these are needed.

File: docker-for-windows/index.md [hunk 3]
```diff
@@ -52,27 +51,46 @@ If you have not already done so, please install Docker for Windows. You can down
...
+    app installed at a time. Also, you will need to save images and export
+    containers you want to keep before uninstalling the current version before
+    installing another. For more about this, see the
+    [FAQs about beta and stable channels](faqs.md#questions-about-stable-and-beta-channels).
 
 ##  What to know before you install
 
 * **README FIRST for Docker Toolbox and Docker Machine users**: Docker for Windows requires Microsoft Hyper-V to run. After Hyper-V is enabled, VirtualBox will no longer work, but any VirtualBox VM images will remain. VirtualBox VMs created with `docker-machine` (including the `default` one typically created during Toolbox install) will no longer start. These VMs cannot be used side-by-side with Docker for Windows. However, you can still use `docker-machine` to manage remote VMs.
-
+<p />
```
Comment (mstanleyjones) [inline docker-for-windows/index.md]: I don't think these are needed either.

File: docker-for-windows/index.md [hunk 4]
```diff
@@ -52,27 +51,46 @@ If you have not already done so, please install Docker for Windows. You can down
...
-
-  >**Note**: If your system does not satisfy these requirements, you can install [Docker Toolbox](/toolbox/overview.md), which uses Oracle Virtual Box instead of Hyper-V.
-
+<p />
+* The Hyper-V package must be enabled for Docker for Windows to work. The Docker for Windows installer will enable it for you, if needed. (This requires a reboot). If your system does not satisfy these requirements, you can install [Docker Toolbox](/toolbox/overview.md), which uses Oracle Virtual Box instead of Hyper-V.
+<p />
 * Virtualization must be enabled. Typically, virtualization is enabled by default. (Note that this is different from having Hyper-V enabled.) For more detail see [Virtualization must be enabled](troubleshoot.md#virtualization-must-be-enabled) in Troubleshooting.
-
-* **What the install includes**: The installation provides [Docker Engine](https://docs.docker.com/engine/userguide/intro/), Docker CLI client, [Docker Compose](https://docs.docker.com/compose/overview/), and [Docker Machine](https://docs.docker.com/machine/overview/).
+<br><br>
```
Comment (mstanleyjones) [inline docker-for-windows/index.md]: I don't think these are needed either.

**Comment by londoncalling** [comment]
LGTM

**Pull request merged**